client/bindings/python/examples/get_raw_block.py

```diff
@@ -0,0 +1,14 @@
+from iota_client import IotaClient, MnemonicSecretManager
+
+# Create an IotaClient instance
+client = IotaClient({'nodes': ['https://api.testnet.shimmer.network']})
+
+# Create and post a block without payload
+block_id = client.build_and_post_block()[0]
```

Comment:
is this test and the test below really necessary to separate?

Reply:
These examples are included in the wiki so it's preferable to clearly separate them so they can be linked independantly